src/sklearn_evaluation/plot/agglomerative.py

```diff
@@ -0,0 +1,219 @@
+import numpy as np
+from numpy.linalg import svd
+from numpy.matlib import repmat
+import matplotlib.pyplot as plt
+
+from scipy.cluster.hierarchy import dendrogram
+from sklearn.cluster import AgglomerativeClustering
+
+from warnings import warn
+from ..telemetry import SKLearnEvaluationLogger
```

Comment:
please change this to an absolute import `from sklearn.telemetry import SKLearnEvaluationLogger`

Reply:
you'll see some relative imports in the codebase but we're moving away from them